### ISSUE TITLE
关于LOSSac

### ISSUE THREAD

**Issue by yumingfan-0219** [wpy1999/BAS#4]
您好！关于您的论文有个小问题：
请问您Area Constraint Loss在代码中哪一部分？我没有找到这个

非常感谢您优秀的工作！

**Comment by wpy1999** [comment]
您好，BAS损失和面积约束损失被定义在了同一个函数中在代码中，如下所示
`def bas_loss(self):
        batch = self.x_sum.size(0)
        x_sum = self.x_sum.clone().detach()
        x_res = self.x_erase_sum
        res = x_res / (x_sum + 1e-8)
        res[x_res>=x_sum] = 0 ## or 1

        x_saliency = self.x_saliency
        x_saliency = x_saliency.clone().view(batch, -1)
        x_saliency = x_saliency.mean(1)  

        loss = res + x_saliency
        loss = loss.mean(0) 
        return loss`
其中x_saliency是面积约束损失，res是背景激活抑制损失。希望对您有帮助。

**Comment by yumingfan-0219** [comment]
我以为会另外定义一个， 所以没看明白，感谢回复
